AUTHORS.txt

```diff
@@ -8,6 +8,7 @@ Jean-Denis Courcol
 Jorge Blanco Alonso
 Matthias Wolf
 Mike Gevaert (maintainer)
+Nadir Roman Guerrero
```

Comment:
I would also add `Ankur Sinha` (#182) and others from:
https://github.com/BlueBrain/libsonata/graphs/contributors

Reply:
I agree we should revise who has contributed. I'd probably make a distinction, though between authors and contributors.

Reply:
Your call! If you need any help, just let me know and I'll be glad, ok?

Reply:
From the contributors graph, only Nadir and Ankur Sinha are missing in this file.
Since @mgeplf is the maintainer and he's the only one who has been updating this file, I leave it to him.

Reply:
Ankur Sinha's contributions have been very helpful, so I'll add them.

Thanks for the updates!

Reply:
(was off, so didn't respond earlier, sorry)